app/services/concerns/author_extractor.rb

```diff
@@ -18,6 +18,6 @@ def author_from_xml(xml)
       acct   = "#{username}@#{domain}"
     end
 
-    FollowRemoteAccountService.new.call(acct)
+    FollowRemoteAccountService.new.call(acct, update_profile: update_profile)
```

Comment:
I might be wrong on the syntax rules because I've never used them like this but isn't that the hash syntax?